keyboards/planck/keymaps/muppetjones/config.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#ifdef AUDIO_ENABLE
+#    define STARTUP_SONG SONG(PLANCK_SOUND)
+// #define STARTUP_SONG SONG(NO_SOUND)
+
+#    define DEFAULT_LAYER_SONGS \
+        { SONG(QWERTY_SOUND), SONG(COLEMAK_SOUND), SONG(DVORAK_SOUND) }
+#endif
+
+/*
+ * MIDI options
+ */
+
+/* Prevent use of disabled MIDI features in the keymap */
+//#define MIDI_ENABLE_STRICT 1
+
```

Comment:
```suggestion
```

Reply:
Leaving this was intentional, and as comment lines aren't included at compile time, it doesn't hurt to leave them in a personal keymap.

Reply:
This define no longer exists.